src/pages/Dashboard/Settings/Filters/Filters.tsx

```diff
@@ -93,28 +97,30 @@ export default function Filters(props: {
 	}, [projects])
 
 	return (
-		<Stack>
+		<section id="date">
```

Comment:
Are ids actually relevant here?

Reply:
Ids on a section is a good practice as it allows page anchor

Reply:
Good to know! But are we using anchors? Well, I don't mind them anyways, but it's good to know.